wallet-new/server/Cardano/Wallet/Server.hs

```diff
@@ -1,36 +1,47 @@
 module Cardano.Wallet.Server
     ( walletServer
+    , walletDevServer
     ) where
 
 import           Servant
 import           Universum
 
 import           Cardano.Wallet.API
 import qualified Cardano.Wallet.API.Development.Handlers as Dev
-import qualified Cardano.Wallet.API.V0 as V0
 import qualified Cardano.Wallet.API.V1.Handlers as V1
+import qualified Cardano.Wallet.API.V1.Swagger as Swagger
 import           Cardano.Wallet.Kernel
 import           Cardano.Wallet.Server.CLI (RunMode (..))
-
+import           Pos.Update.Configuration (HasUpdateConfiguration, curSoftwareVersion)
+import           Pos.Util.CompileInfo (HasCompileInfo, compileInfo)
+import           Servant.Swagger.UI (swaggerSchemaUIServer)
 
 -- | Serve the REST interface to the wallet
 --
 -- NOTE: Unlike the legacy server, the handlers will not run in a special
 -- Cardano monad because they just interfact with the Wallet object.
-walletServer :: ActiveWallet
-             -> RunMode
+walletServer :: (HasCompileInfo, HasUpdateConfiguration)
+             => ActiveWallet
```

Comment:
As this is @edsko 's territory mainly, I just wanted to check if he's happy with the (even if temporary) signature change, as I know he's working hard to have as less "legacy baggage" as possible for the new handlers 😁